server/datastore/mysql/apple_mdm.go

```diff
@@ -2176,46 +2184,7 @@ func (ds *Datastore) bulkSetPendingMDMAppleHostProfilesDB(
 	profilesToInsert := make(map[string]*fleet.MDMAppleProfilePayload)
 
 	executeUpsertBatch := func(valuePart string, args []any) error {
-		// Check if the update needs to be done at all.
```

Comment:
Why is all this removed?

Reply:
I shouldn't have added this in the first place. And we had a Slack discussion about this in regards to issues seen in load test.

We already identify the profiles that need updates above https://github.com/fleetdm/fleet/blob/d98c34f823883b4c014bb69d69bf3e93fa37089f/server/datastore/mysql/apple_mdm.go#L1990-L2009 and this extra DB call and check doesn't (and shouldn't) add any value.